server/s3_put_multiobject_copy_action.cc

```diff
@@ -279,15 +279,7 @@ bool S3PutMultipartCopyAction::copy_part_object_cb() {
 // TODO: Add this function to a common class
 void S3PutMultipartCopyAction::create_part_object() {
   s3_log(S3_LOG_INFO, stripped_request_id, "%s Entry\n", __func__);
-  if (additional_object_metadata->get_number_of_fragments() == 0) {
-    create_part();
-  } else {
-    // Incase of fragments or multipart upload
-    // each part copy needs separate object creation
-    s3_copy_part_action_state = S3PutObjectActionState::validationFailed;
-    set_s3_error("NotImplemented");
-    send_response_to_s3_client();
-  }
+  create_part();
```

Comment:
We can now merge `create_part()` and `create_part_object()`

Reply:
Done